.eslintrc.js

```diff
@@ -66,9 +66,33 @@ module.exports = {
           "tsx": "never"
         }],
 
+        // TODO: Partially implemented
+        '@typescript-eslint/naming-convention': [
+          'error',
+          {
+            selector: 'typeLike',
+            format: ['PascalCase'],
+          },
+          {
+            selector: 'enumMember',
+            format: ['PascalCase'],
+          },
+          {
```

Comment:
@undsoft LGTM. except for this duplicated rule.

Reply:
True, fixed.